rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml

```diff
@@ -0,0 +1,34 @@
+title: PowerShell without PowerShell.exe
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects loading Powershell packet by non-Powershell process
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tag:
+    - attack.defense_evasion
```

Comment:
Hello @nsaddler! 

Thank you for your contribution! 
I believe that you can add a technique tag for PowerShell execution — `attack.t1059.001`